data/json/items/armor/torso_armor.json

```diff
@@ -2178,7 +2178,7 @@
     "type": "ARMOR",
     "copy-from": "lc_mirror_armor",
     "name": { "str": "tempered steel mirror armor", "str_pl": "sets of tempered steel mirror armor" },
-    "description": "Mirror armor is either round or square plates connected by fasteners to cover all four sides of the vital areas of the torso, mirror plate was usually worn on top of other armor like chainmail or brigandines.  This set is made of intricate alternating circular and rectangular tempered steel plated, detailed on the outsides and polished to a mirrored sheen in the center.",
+    "description": "Mirror armor earns its name from its distinct shape and polished sheen.  Round and rectangular steel plates are fastened together to protect all four sides of the torso.  It can be worn on top of other armor like chainmail and brigandines.  Masterfully crafted with tempered steel, this set features a prominent circular plate on the front that has been polished to a mirrorlike sheen, flanked with supporting plates intricately joined to fit the body, all painstakingly decorated.  Will protect you from serious harm, as well as from the evil gazes of the undead.",
```

Comment:
Perhaps you could change "painstakingly" to "intricately"? Just a minor suggestion.

Reply:
There's already "intricately" in the same sentence; I'd go with "meticulously" instead of "painstakingly".

Reply:
Yeah, that would sound better. Thanks for catching that.

Reply:
Good suggestion